R/fb_plot_number_species_by_trait.R

```diff
@@ -0,0 +1,82 @@
+#' Show Number of Species per Trait
+#'
+#' @inheritParams fb_get_coverage
+#'
+#' @param threshold_species_number `NULL` or `numeric(1)` \[default = `NULL`\]\cr{}
+#'        If `NULL` doesn't show a line otherwise adds a vertical line
+#'        at the specified number of species
+#'
+#' @return a ggplot2 object
+#'
+#' @examples
+#' data(species_traits)
+#' 
+#' \dontrun{%
```

Comment:
From experience, it may create a bug in some cases if it doesn't, but you're right, I should probably delete it. I think it's more a LaTeX habit than an R one.